docs/agent-api.asciidoc

```diff
@@ -837,3 +837,13 @@ the node process is terminated automatically when an uncaught exception has been
 https://nodejs.org/api/process.html#process_event_uncaughtexception[It is recommended] that you don't leave the process running after receiving an uncaught exception,
 so if you are using the optional callback,
 remember to terminate the node process.
+
+[[apm-flush]]
+==== `apm.flush()`
+
+[source,js]
+----
+apm.flush()
+----
+
+Flush the transaction queue immediately.
```

Comment:
We should probably underline that this means that the transactions are sent to the APM Server and optionally also add a bit about how to configure it, e.g.

```asciidoc
Manually flush the in-memory transaction queue and send all the transactions to the APM Server.
The queue is otherwise flushed automatically,
controlled by the <<flush-interval,`flushInterval`>> and/or <<max-queue-size,`maxQueueSize`>> config options.
```